dash/frontend/src/app/modules/private/pages/cluster/template-constraint-manifest/template-constraint-manifest.component.ts

```diff
@@ -45,13 +50,19 @@ export class TemplateConstraintManifestComponent implements OnInit {
 
   ngOnInit(): void {}
 
-  setEditorContent($event: any) {
-  }
+  setEditorContent($event: any) {}
 
   saveManifest() {
     this.dialogRef.close({manifestData: JSON.parse(this.rawTemplate)});
   }
 
+  onCopyClick() {
```

Comment:
It looks lie you're using the copy to clipboard component in the template, so is this code actually used?

Reply:
Removed